shell/common/gin_converters/content_converter.cc

```diff
@@ -7,9 +7,10 @@
 #include <string>
 #include <vector>
 
+#include "content/public/browser/context_menu_params.h"
 #include "content/public/browser/native_web_keyboard_event.h"
 #include "content/public/browser/web_contents.h"
-#include "content/public/common/context_menu_params.h"
+#include "content/public/common/untrustworthy_context_menu_params.h"
```

Comment:
what is this header needed for ?

Reply:
content::CustomContextMenuContext got moved there